src/main/java/com/cinema/model/ShoppingCart.java

```diff
@@ -0,0 +1,53 @@
+package com.cinema.model;
+
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    @OneToOne
+    private User user;
```

Comment:
I think we need to use cascades here. Can User exist if the shopping cart is deleted? Do we need to store tickets if the shopping cart is deleted?

Reply:
I guess it all depends on whether we want to keep the information about the sold tickets and user's carts in the DB for future reports or what not. I prefer to keep the info until we implement the logic for deleting users.

Reply:
But they are not sold yet, it is a shopping cart, not order

Reply:
> I think we need to use cascades here. Can User exist if the shopping cart is deleted? Do we need to store tickets if the shopping cart is deleted?

if we put cascade over this field user will be deleted if cart is deleted